<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametryBadaniaDlg</class>
 <widget class="QDialog" name="ParametryBadaniaDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParametryBadania</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="error">
     <property name="styleSheet">
      <string notr="true">color:red;font-size:12pt;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="ParametryBadaniaOgolneDlg" name="tabBadanie">
      <attribute name="title">
       <string>Badanie</string>
      </attribute>
     </widget>
     <widget class="ParametryBadaniaCzujkiDlg" name="tabCzujki">
      <attribute name="title">
       <string>Czujki</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>418</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbCancel">
        <property name="text">
         <string>Anuluj</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbOK">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParametryBadaniaOgolneDlg</class>
   <extends>QWidget</extends>
   <header>parametrybadaniaogolnedlg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ParametryBadaniaCzujkiDlg</class>
   <extends>QWidget</extends>
   <header>parametrybadaniaczujkidlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
